.make/test.mk

```diff
@@ -187,6 +193,22 @@ test-remote-no-coverage: prebuild-check $(SOURCES)
 test-migration: prebuild-check migration/sqlbindata.go migration/sqlbindata_test.go
 	F8_RESOURCE_DATABASE=1 F8_LOG_LEVEL=$(F8_LOG_LEVEL) go test $(GO_TEST_VERBOSITY_FLAG) github.com/fabric8-services/fabric8-wit/migration
 
+.PHONY: test-e2e
+## Runs the end-to-end tests WITHOUT producing coverage files for each package.
+test-e2e: prebuild-check deps docker-compose-up $(SOURCES)
+	$(call log-info,"Running tests: $@")
+	# ## Clone the fabric8-test repo
+	@if [ "$(FABRIC8_TEST_DIR)" ]; then \
+		echo "NOT Removing existing fabric8-test dir $(FABRIC8_TEST_DIR)"; \
+		rm -rf $(FABRIC8_TEST_DIR); \
```

Comment:
@rgarg1 What's with the message above this `rm` call? It says that it's not removing the DIR and then you're removing it.

Reply:
@kwk That is a remnant of the log I placed while testing. Will remove this message.